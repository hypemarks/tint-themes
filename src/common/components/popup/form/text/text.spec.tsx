import { render } from '@testing-library/react';
import React from 'react';
import Text from './text';

describe('Test <Text /> Component', () => {
  it('should render snapshot', () => {
    const wrapper = render(
      <Text value={'value'} name={'name'} onChange={jest.fn} id={'id'} onBlur={jest.fn} type={'type'} />
    );
    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
