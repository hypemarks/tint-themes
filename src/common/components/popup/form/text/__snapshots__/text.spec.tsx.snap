// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Text /> Component should render snapshot 1`] = `
<DocumentFragment>
  <input
    class="undefined text"
    id="id"
    name="name"
    placeholder="Type here..."
    type="type"
    value="value"
  />
</DocumentFragment>
`;
